include/rdm/defs.h

```diff
@@ -4,6 +4,7 @@
  *  - File renamed to rdm/defs.h
  *  - BSR E1.37-7 definitions added
  *  - clang-format off directive added
+ *  - E1.33 definitions added
```

Comment:
We should say "ANSI E1.33" and "ANSI E1.37-7" (change from BSR to reflect that the standard has been published)